examples/MerminMagic/main.jl

```diff
@@ -88,44 +84,25 @@ function main()
     # j is the column number given to Bob
     println("Dealer: Let's Play Mermin's Magic Square Game!")
 
-    println("Please enter a number in the range [1,3] as row index")
-    i = parse(Int64,readline())
-    println("Please enter a number in the range [1,3] as col index")
-    j = parse(Int64,readline())
-
-    # test to make sure in bound
-    while i > 3 || i < 1
-        println("Please be nice and enter a number in the range [1,3]")
-        i = parse(Int64,readline())
-    end
-
-    while j > 3 || j < 1
-        println("Please be nice and enter a number in the range [1,3]")
-        j = parse(Int64,readline())
-    end
+    i = input_idx("row")
+    j = input_idx("column")
 
     println("Alice will provide three numbers to be written on row: $(i)")
     println("Bob will provide three numbers to be written on column: $(j)")
 
-    # initialize quantum registero
-    reg1 = zero_state(4)
-
-    init_double_bell_state(reg1)
+    # initialize register to two pairs of bell state
+    reg1 = init_double_bell_state()
 
     # do measurement and get answer string
-    # I sort of cheated here, since we know Alice Bob
-    # need to adhere to the rules, the third measurement
-    # won't be necessary hence I just used the fact
-    # that a3 = 1 * a1 * a2 and b3 = -1 * b1 * b2
-    a1, a2, b1, b2 = meas_by_idx(reg1, i, j)
-
-    println("Alice should give answer: $a1 , $a2 , $(1*a1*a2)")
-    println("Bob should give answer: $b1 , $b2 , $(-1*b1*b2)")
-    println("We should see $([a1,a2,1*a1*a2][j]) is equal to $([b1,b2,-1*b1*b2][i])")
+    a1, a2, a3, b1, b2, b3 = meas_by_idx(reg1, i, j)
+
+    println("Alice should give answer: $a1 , $a2 , $a3")
+    println("Bob should give answer: $b1 , $b2 , $b3")
+    println("We should see $([a1,a2,a3][j]) is equal to $([b1,b2,b3][i])")
```

Comment:
Imagine I am a user using your code. If I print this line, I will get useless information: "We should see 1 is equal to 1".